ixofoundation/ixo-webclient
# chore(refactor): bonds module redux

Work done

- https://nonaredteam.atlassian.net/browse/IXO-119

Refactored redux for the bonds

Steps to test:

- delete `node_modules`
- `npm install`
- `npm run test`

run app and see that bonds section works
